code/lib/builder-vite/src/vite-config.test.ts

```diff
@@ -0,0 +1,66 @@
+import type { Options, Presets } from '@storybook/types';
+import { loadConfigFromFile } from 'vite';
+import { commonConfig } from './vite-config';
+
+jest.mock('vite', () => ({
+  ...jest.requireActual('vite'),
+  loadConfigFromFile: jest.fn(async () => ({})),
+}));
+const loadConfigFromFileMock = jest.mocked(loadConfigFromFile);
+
+const dummyOptions: Options = {
+  configType: 'DEVELOPMENT',
+  configDir: '/tmp',
+  packageJson: {},
+  presets: {
+    apply: async (key: string) =>
+      ({
+        framework: {
+          name: '',
+        },
+        addons: [],
+        core: {
+          builder: {},
+        },
```

Comment:
What if you add `options: {}` here?  Maybe that would fix the test?

Reply:
Yes that works. Thanks for the idea! :-)